### PR TITLE
add documentation for non-resource requests

### PR DESCRIPTION
**Pull request by snamoah**
#### Pre-Submission Checklist

- [x] Your pull request targets the `master` branch of Kudobuzz.
- [x] Branch name is descriptive. 

#### Checklist:

- [x] Improved documentation or readme where necessary. 
- [x] Your commits messages format follows our commit message guideline.

#### What does this pr do?
Add documentation for non-resource requests

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#### PreSubmission Checklist
```
Comment (obengwilliam) [inline .github/ISSUE_TEMPLATE.md:1]: this does not need and issue template or  ?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#### Pre-Submission Checklist
+<!-- Go over all points below, and after creating the PR, tick all the checkboxes that apply. -->
+<!-- All points should be verified, otherwise, read the CONTRIBUTING guidelines from above-->
+<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+<!--You also read mall from https://github.com/kudobuzz/guides/edit/master/code-review-guidelines.md-->
+
+- [ ] Your pull request targets the `stage` branch of Kudobuzz.
+- [ ] Branch name is descriptive. 
+- [ ] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.
```
Comment (obengwilliam) [inline .github/PULL_REQUEST_TEMPLATE.md:9]: check comment above

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (snamoah) [reply .github/ISSUE_TEMPLATE.md:1]: ohk... so what about PR template?

Comment (snamoah) [reply .github/ISSUE_TEMPLATE.md:1]: and there will be no format when submitting PRs?

Comment (obengwilliam) [reply .github/ISSUE_TEMPLATE.md:1]: yeah i think its fine in this case

Comment (obengwilliam) [reply .github/ISSUE_TEMPLATE.md:1]: ok lets leave it

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (obengwilliam) [reply .github/PULL_REQUEST_TEMPLATE.md:9]: ok let's leave it

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (snamoah) [reply .github/ISSUE_TEMPLATE.md:1]: ok

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (snamoah) [reply .github/PULL_REQUEST_TEMPLATE.md:9]: ok

File: README.md [hunk 3]
```diff
@@ -45,6 +45,8 @@ This document borrows heavily from:
 * Formats should be in the form of api/v2/resource/{id}
 * Design resource representations. Don’t simply represent database tables.
 * Support field projections on resources. Allow clients to reduce the number of fields that come back in the response.
+* For non-resource requests, url should identify the action being taken. E.g. if the endpoint converts 100 euros to chinese yen, is should look like `/convert?from=EURO&to=YEN&amount=100`
+* Make it clear in your API documentation that the "non-resource" scenarios are different
```
Comment (ghost) [inline README.md]: @snamoah Please how do you make it clear that "non-resource" scenarios are different in your API. Can you site an example?

And can you give an example of an endpoint that accepts non-resource POST requests if possible?

**Review by ghost — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (snamoah) [reply README.md]: @richardkdbz the example exists in the guidelines. is it not clear enough?

Comment (ghost) [reply README.md]: @snamoah I mean example on how you make it clear that "non-resource" scenarios are fiferent.

Comment (snamoah) [reply README.md]: @richardkdbz updated

**Pull request merged**